server/grpc/grpc.go

```diff
@@ -363,7 +363,17 @@ func (g *grpcServer) processRequest(stream grpc.ServerStream, service *service,
 					err = errors.InternalServerError("go.micro.server", "panic recovered: %v", r)
 				}
 			}()
-			returnValues = function.Call([]reflect.Value{service.rcvr, mtype.prepareContext(ctx), reflect.ValueOf(argv.Interface()), reflect.ValueOf(rsp)})
+			vreq := reflect.ValueOf(argv.Interface())
+			if vreq.IsNil() {
```

Comment:
Surely if the argument is nil this is not an issue because its just a blank arg?